spec/services/acuant/acuant_client_spec.rb

```diff
@@ -48,25 +48,50 @@
   end
 
   describe '#post_images' do
-    it 'sends an upload image request for the front image' do
-      instance_id = 'this-is-a-test-instance-id'
-      url = URI.join(
+    let(:instance_id) { 'this-is-a-test-instance-id' }
+    let(:image_upload_url) do
+      URI.join(
         Figaro.env.acuant_assure_id_url, "/AssureIDService/Document/#{instance_id}/Image"
       )
-      stub_request(:post, url).with(query: { side: 0, light: 0 }).to_return(body: '', status: 201)
-      stub_request(:post, url).with(query: { side: 1, light: 0 }).to_return(body: '', status: 201)
-      results_url = URI.join(
+    end
+    let(:front_image_query) { { query: { side: 0, light: 0 } } }
+    let(:back_image_query)  { { query: { side: 1, light: 0 } } }
+    let(:results_url) do
+      URI.join(
         Figaro.env.acuant_assure_id_url, "/AssureIDService/Document/#{instance_id}"
       )
+    end
+    let(:get_face_image_url) do
+      URI.join(
+        Figaro.env.acuant_assure_id_url,
+        "/AssureIDService/Document/#{instance_id}/Field/Image?key=Photo",
+      )
+    end
+    let(:facial_match_url) { URI.join(Figaro.env.acuant_facial_match_url, '/api/v1/facematch') }
+    let(:liveness_url) { URI.join(Figaro.env.acuant_passlive_url, '/api/v1/liveness') }
+
+    it 'sends an upload image request for the front, back, and selfie images' do
```

Comment:
Do we need to enable liveness for this spec? It looks like otherwise it ignores the selfie.

I noticed the gap in the coverage above. Maybe a test case for this w/ selfie enabled and selfie disabled?